metadata-db-app/int-test/cmr/metadata_db/int_test/utility.clj

```diff
@@ -330,8 +339,15 @@
   [concept]
   (let [{:keys [concept-id revision-id]} concept
         stored-concept (:concept (get-concept-by-id-and-revision concept-id revision-id))]
-    (is (= (expected-concept concept)
-           (dissoc stored-concept :revision-date :transaction-id :created-at)))))
+    (if (= :variable (:concept-type concept))
+      ;; We added :coll-concept-id in the attribute in the tests, it'll appear in both concept and
+      ;; stored-concept in memory db, but not in the stored-concept in real db. This is the cas
```

Comment:
minor typo, cas instead of case

Reply:
Thanks. Will change the typo in another ticket so that I don't have to wait for the long build.